x/bonds/client/rest/tx.go

```diff
@@ -444,3 +467,36 @@ func withdrawShareRequestHandler(clientCtx client.Context) http.HandlerFunc {
 		tx.WriteGeneratedTxResponse(clientCtx, w, req.BaseReq, msg)
 	}
 }
+
+type withdrawReserveReq struct {
+	BaseReq      rest.BaseReq `json:"base_req" yaml:"base_req"`
+	BondDid      string       `json:"bond_did" yaml:"bond_did"`
+	Amount       string       `json:"amount" yaml:"amount"`
+	RecipientDid string       `json:"recipient_did" yaml:"recipient_did"`
```

Comment:
(Improvement) Maybe this should be renamed to `WithdrawerDid` for consistency?

Reply:
Definitely